python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/qos.j2

```diff
@@ -24,6 +24,10 @@ QOS rewrite DSCP: **disabled**
 
 {{ ecn_command }}
 {%     endif %}
+{%     if qos.tx_queue.shape_rate_percent_adaptive is arista.avd.defined(true) %}
+
+QOS txQueue percentage-based allocation: **enabled**
```

Comment:
@ctyrider I know this is the name of the TOI but I feel like it fails to explain that it is based on the parent's available bandwidth and not on the parent's link speed. Can we make this description more informative?

The TOI states: _Here adaptive keyword means to use the parent interface's available bandwidth._ and our description does not even mention `adaptive` maybe it should be something like this: 

```suggestion
Adaptive txQueue percentage-based allocation: **enabled**
```